key.ui/src/main/java/de/uka/ilkd/key/gui/ExampleChooser.java

```diff
@@ -241,10 +241,6 @@ private ExampleChooser(File examplesDir) {
         exampleList.addMouseListener(new MouseAdapter() {
             @Override
             public void mouseClicked(MouseEvent e) {
-                // select closest tree entry to ensure that double clicks
-                // always open the entry the double click was performed on
-                TreePath closest = exampleList.getClosestPathForLocation(e.getX(), e.getY());
-                exampleList.setSelectionPath(closest);
```

Comment:
Why is this removed?

Reply:
I didn't find any positive case where this actually does anything.

Other than that, as I stated, double-clicking on a folder instantly loads a previously hidden example when the list is repositioned (e.g. when fully scrolled down).  The list is expanded, repositioned, then the event handler is catched and counts the double click on a folder as a double click on the repositioned list which is completely wrong.

Reply:
This fix does something if the user clicks left of an entry (hitting the background of the tree view). In that case, the code would load the last selected example and not the one the user actually double clicked at.

Reply:
Ah I see the problem, good catch.

The problem is that a double click *anywhere* in the tree leads to a load. I pushed a fix to check whether an actual item was doubleclicked. Could you check if it works as intended and close this conversation, please?

Reply:
Thank you. The fix works as expected.